tests/dir_helpers.py

```diff
@@ -285,3 +293,21 @@ def erepo_dir(tmp_path_factory, monkeypatch):
         path.dvc.close()
 
     return path
+
+
+@pytest.fixture
+def external_git_dir(tmp_path_factory, monkeypatch):
```

Comment:
Making a new fixture for a single test is an overkill, doing some copy-paste inside is even worse. You may simply use `erepo_dir` and remove `.dvc` dir there.